File: responses/Z9S2T8-8XQ/3_find_maximum_product.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(X, B) {
    if(B.length == 0 || X > B.length)
        return 0
    
    var prod = 1
    
    for(i = B.length - 1; i >= B.length - X; i--) {
        prod = prod * B[i]
    }
    
    return prod
}

File: responses/HFJPHB-FJN/3_find_maximum_product.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X, B):
    # write your code in Python
    if len(B) == 0 : return 0
    if X > len(B) : return 0
    
    B.sort()

    prod = 1
    for i in B[len(B) - X:]:
        prod *= i
        
    return prod
    
    